app/models/krikri/original_record.rb

```diff
@@ -8,13 +8,17 @@ class OriginalRecord
     attr_writer :content_type
 
     def initialize(identifier)
+      raise ArgumentError, "#{identifier} is an invalid local name" if
+        identifier.include?('/')
       @local_name = identifier
     end
 
```

Comment:
It might be nice to include a comment explaining when to instantiate an instance OriginalRecord vs. when to interact with OriginalRecord as a class.  It was very helpful when you explained it to me.